File: ServerSide/SAMS.BusinessLogic/Models/Faculty/FacultyModel.cs
using SAMS.BusinessLogic.Models.StudyProgrammes;
using System;
using System.Collections.Generic;
using System.Text;

namespace SAMS.BusinessLogic.Models.Faculty
{
    public class FacultyModel
    {
        public int Id { get; set; }
        public string Code { get; set; }

        public string TitleLv { get; set; }
        public string TitleRu { get; set; }
        public string TitleEn { get; set; }

        public string ShortTitleLv { get; set; }
        public string ShortTitleRu { get; set; }
        public string ShortTitleEn { get; set; }

        public List<StudyProgrammeModel> StudyProgrammes { get; set; }
    }
}

File: ClientSide/src/app/pages/faculties/models/facultyModel.ts
import { StudyProgrammeModel } from './studyProgrammeModel';

export class FacultyModel {

    public id: number;
    public code: string;

    public titleEn: string;
    public titleLv: string;
    public titleRu: string;
    
    public shortTitleEn: string;
    public shortTitleLv: string;
    public shortTitleRu: string;
    
    public studyProgrammes: Array<StudyProgrammeModel>;

    constructor(
        id: number,
        code: string,
        titleEn: string,
        titleLv: string,
        titleRu: string,
        shortTitleEn: string,
        shortTitleLv: string,
        shortTitleRu: string,
        studyProgrammes: Array<StudyProgrammeModel>
    ) {
        this.id = id;
        this.code = code;
        this.titleEn = titleEn;
        this.titleLv = titleLv;
        this.titleRu = titleRu;
        this.shortTitleEn = shortTitleEn;
        this.shortTitleLv = shortTitleLv;
        this.shortTitleRu = shortTitleRu;
        this.studyProgrammes = studyProgrammes;
    }

}
